web/tests/dispatch_subs.test.js

```diff
@@ -236,6 +240,8 @@ test("stream delete (special streams)", ({override}) => {
     const event = event_fixtures.stream__delete;
 
     for (const stream of event.streams) {
+        stream.archived = true;
+        stream.subscribed = true;
```

Comment:
I don't completely understand this whole testcase.
Please be a bit more thorough during review.

Reply:
Currently, deactivating/archiving a stream involves updating the `archived: true` and `subscribed: false` in the stream objects. 

In the given testcase, `test("stream delete (normal)")` precedes the modification of values(`archived` and `subscribed`) in `event_fixtures.stream__delete`.

Consequently, for `test("stream delete (special streams)")`, where `remove_sidebar_row` is expected to be called, the streams in `event_fixtures.stream__delete` need to be marked as `archived: false` and `subscribed: true`, as remove_sidebar_row is only triggered for `subscribed: true` streams.

Here:
```
if (was_subscribed) {
   stream_list.remove_sidebar_row(stream.stream_id);
}
```

But I see how I have wrongly written `archived: true`. I will be more careful.